packages/gatsby-theme-docs/src/modules/ai-assistant/components/ai-assistant-launch-button.tsx

```diff
@@ -13,9 +13,20 @@ const AssistantLaunchContainer = styled.div`
     }
   }
 `;
+interface MessageHistory {
```

Comment:
we tend to use type rather than interfaces for types and props types definition

Reply:
Updated